src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/integration/sentencing/SentencingMigrationIntTest.kt

```diff
@@ -45,6 +48,14 @@ class SentencingMigrationIntTest : SqsIntegrationTestBase() {
   @Autowired
   private lateinit var migrationHistoryRepository: MigrationHistoryRepository
 
+  @BeforeEach
+  fun cleanQueue() {
+    awsSqsSentencingMigrationClient.purgeQueue(PurgeQueueRequest.builder().queueUrl(sentencingMigrationUrl).build()).get()
+    awsSqsSentencingMigrationDlqClient?.purgeQueue(PurgeQueueRequest.builder().queueUrl(sentencingMigrationDlqUrl).build())?.get()
+    await untilCallTo { awsSqsSentencingMigrationClient.countAllMessagesOnQueue(sentencingMigrationUrl).get() } matches { it == 0 }
+    await untilCallTo { awsSqsSentencingMigrationDlqClient?.countAllMessagesOnQueue(sentencingMigrationDlqUrl!!)?.get() } matches { it == 0 }
+  }
+
```

Comment:
Might be nice to push this into the base class and accept a list of queues to clean?